ggml.c

```diff
@@ -20283,7 +20284,22 @@ struct gguf_context * gguf_init_empty(void) {
 }
 
 struct gguf_context * gguf_init_from_file(const char * fname, struct gguf_init_params params) {
-    FILE * file = fopen(fname, "rb");
+#ifdef _WIN32
+    setlocale(LC_ALL, "");
```

Comment:
I think it is important to restore the previous locale before returning. This breaks the markdown printer `llama-bench`, which expects the locale to be set to utf-8.

Reply:
Good catch, this should be handled now